collect_app/src/main/java/org/odk/collect/android/fragments/DataManagerList.java

```diff
@@ -149,7 +143,7 @@ protected void onResume() {
     }
 
     @Override
-    protected void onPause() {
```

Comment:
Ah I just checked the Fragment class and you're right. It is public. I was looking at the FragmentActivity earlier.